generators/server/templates/quarkus/src/main/resources/application.properties.ejs

```diff
@@ -96,7 +96,18 @@ quarkus.oidc.authentication.cookie-path=/
 quarkus.oidc.authentication.redirect-path=/login/oauth2/code/oidc
 quarkus.oidc.authentication.restore-path-after-redirect=false
 
+
+jhipster.oidc.logout-url=http://localhost:9080/auth/realms/jhipster/protocol/openid-connect/logout
+
+quarkus.http.auth.permission.public.paths=/management/prometheus, /management/prometheus/*, /management/health, /management/info, /
```

Comment:
We should be cautious about exposing `/management/health` to public as it can leak sensitive inner component details.